jerry-core/parser/js/js-parser-expr.c

```diff
@@ -633,6 +633,28 @@ parser_parse_class (parser_context_t *context_p, /**< context */
 } /* parser_parse_class */
 #endif /* !CONFIG_DISABLE_ES2015_CLASS */
 
+#ifndef CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER
```

Comment:
Is this feature turned on by default in ES5.1 profile?

Reply:
No. It is the opposite.

Reply:
Please add a description about the new feature to [the profile documentation](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/profiles/README.md).

Reply:
The CONFIG_DISABLE_ES2015_OBJECT_INITIALIZER is not a new macro, just a new feature is included into it.